src/subdomains/core/liquidity-management/adapters/balances/blockchain.adapter.ts

```diff
@@ -166,7 +172,7 @@ export class BlockchainAdapter implements LiquidityBalanceIntegration {
     );
   }
 
-  private async getForEvmAsset(assets: Asset[], client: EvmClient): Promise<void> {
+  private async getForTokenAsset(assets: Asset[], client: TokenClient): Promise<void> {
```

Comment:
Maybe it should rather be `getForTokenChain` or something? It also fetches coin, not only token balance.